pkg/imgutil/filtering_test.go

```diff
@@ -20,8 +20,9 @@ import (
 	"testing"
 	"time"
 
-	"github.com/containerd/containerd/v2/core/images"
```

Comment:
Unrelated.
I believe I fixed this in another recent PR on main, so, shouldn't be a concern?